app/models/review.rb

```diff
@@ -0,0 +1,17 @@
+class Review < ActiveRecord::Base
+  after_save :update_video_rating!
+
+  belongs_to :creator, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :video
+
+  validates_presence_of :user_id
+  validates_presence_of :video_id
+  validates_inclusion_of :rating, in: 1..5
+  validates :body, length: { minimum: 10 }
+
+  private
+
+  def update_video_rating!
+    video.save if video
```

Comment:
What i am trying to do here is every time a review is created or updated (:after_save), the associated video rating should be updated. Video rating is updated every time a video is saved (:before_validation callback). I tried to set autosave: true on the Review's belongs_to: video association, but that's not triggering the associated video to save.